world/admin.py

```diff
@@ -2,8 +2,10 @@
 from embed_video.admin import AdminVideoMixin
 from solo.admin import SingletonModelAdmin
 ######################################################
-# Replace the next line with generated adminModelImports
-from .models import TextSnugget, EmbedSnugget, SnuggetSection, SnuggetSubSection, Infrastructure, InfrastructureGroup, InfrastructureCategory, RecoveryLevels, Location, SiteSettings
```

Comment:
It might be better to leave this in and replace it in the code generator. If we take it out completely, the app won't run empty.